deployment/SampleClaimsProvider/Controllers/ClaimsStoreController.cs

```diff
@@ -0,0 +1,66 @@
+﻿// ---------------------------------------------------------------------------------
+// <copyright company="Microsoft">
+//   Copyright (c) {Microsoft} Corporation. All rights reserved.
+// </copyright>
+// ---------------------------------------------------------------------------------
+
+namespace SampleClaimsProvider
+{
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+
+    [Route("api/claimsstore")]
+    [ApiController]
+    public class ClaimsStoreController : ControllerBase
+    {
+        private readonly IClaimsProviderStore claimsProviderStore;
+        private readonly ILogger<ClaimsStoreController> logger;
+
+        public ClaimsStoreController(ILogger<ClaimsStoreController> logger, IConfiguration configuration, IClaimsProviderStore claimsProviderStore)
+        {
+            this.logger = logger;
+            this.claimsProviderStore = claimsProviderStore;
+        }
+
+        /// <summary> This API adds a specified claim.
+        /// <example>
+        /// [POST] https://{hostname}/api/claimsstore
+        ///     <code>
+        ///     Json Body:
+        ///     {"vehicleId": "testVehicle1", "serviceId": "service1", "claims": [{"name": "claim1","values": [{"value": "value1"}]}]}
+        ///     </code>
+        /// </example>
+        /// </summary>
+        [HttpPost]
+        public async Task<IActionResult> AddClaims([FromBody] ClaimsStoreRequest claimsRequest)
+        {
+            await this.claimsProviderStore.CreateClaimsAsync(claimsRequest.VehicleId, claimsRequest.UserId, claimsRequest.ServiceId, claimsRequest.Claims);
+
+            const string responseMessage = "Added Claims info object.";
+            this.logger.LogInformation("[{tag}] Created claims for: {claimsRequest}", "", claimsRequest);
```

Comment:
Can we remove the use of the empty `[{tag}]` here and elsewhere? From the customer perspective it doesn't make sense why we would have that.